spark/src/main/java/org/apache/zeppelin/spark/SparkInterpreter.java

```diff
@@ -1025,4 +1129,119 @@ public ZeppelinContext getZeppelinContext() {
   public SparkVersion getSparkVersion() {
     return sparkVersion;
   }
+
+  private Object invokeMethod(Object o, String name) {
+    return invokeMethod(o, name, new Class[]{}, new Object[]{});
+  }
+
+  private Object invokeMethod(Object o, String name, Class [] argTypes, Object [] params) {
```

Comment:
It's awesome, you did a lot of work to get both Scala 2.10 and 2.11 supported!

Probably a nit, but how do you think, shall these guys be pulled up to some common ancestor to avoid duplication between `DepInterpreter` and  `SparkInterpreter`? Or may be just extracted and re-used?